packages/matchbox/src/components/Panel/tests/Panel.test.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import Panel from '../Panel';
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
I think we are actually moving away from shallow mounting on the app team - since the components that make up another component are technically implementation details. What was your thinking here?